requirements.txt

```diff
@@ -1,8 +1,8 @@
 parsedatetime==2.1
-gevent==1.1.1
-greenlet==0.4.9
+gevent==20.12.1
```

Comment:
They've jumped from v1 to v20 directly: https://pypi.org/project/gevent/#history
Hence not as major a change.